public/index.html

```diff
@@ -3,24 +3,81 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie</title>
-  </head>
-  <body>
-    <h1>Hello from Hoodie!</h1>
+    <style>
+    body{
+background-color:#FCFCFC;
+animation-name: graytocolorimg;
+animation-duration: 4s;
```

Comment:
Can you explain your thinking behind the animation? I think I’d prefer to keep it very plain and simple, but maybe I’m missing something?